lib/plugins/aws/lib/naming.test.js

```diff
@@ -94,10 +94,16 @@ describe('#naming()', () => {
   });
 
   describe('#getStackName()', () => {
-    it('should use the service name and stage from the service and config', () => {
+    it('should use the service name and stage from the service and config as default', () => {
       serverless.service.service = 'myService';
       expect(sdk.naming.getStackName()).to.equal(`${serverless.service.service}-${options.stage}`);
     });
+
+    it('should use the provider options to supply a custom stackName', () => {
```

Comment:
Hey @ltvolks yes, I think you're right. But I think another test might be good as well so that we still have test coverage in place when this functionality is dropped.